packages/flutter_tools/lib/src/asset.dart

```diff
@@ -268,6 +268,15 @@ class ManifestAssetBundle implements AssetBundle {
     final DevFSStringContent licenses = DevFSStringContent(licenseResult.combinedLicenses);
     additionalDependencies = licenseResult.dependencies;
 
+    if (wildcardDirectories.isNotEmpty) {
```

Comment:
Could a trace message be helpful here?

Reply:
Yeah, if someone is curious why everything is re-running I should provide some context

Reply:
Added and linked to the issue so that someone that is curious and investigating can find the right place to discuss (hopefully)